applications/IgaApplication/custom_elements/shell_5p_element.cpp

```diff
@@ -199,8 +199,8 @@ namespace Kratos
         rKin.dtd1 = prod(rVar.P, rKin.dtd1);
         rKin.dtd2 = prod(rVar.P, rKin.dtd2);
 
-        rKin.metricChange[0] = inner_prod(rKin.A1, rKin.dud1) + 0.5 * norm_2_square(rKin.dud1);
-        rKin.metricChange[1] = inner_prod(rKin.A2, rKin.dud2) + 0.5 * norm_2_square(rKin.dud2);
+        rKin.metricChange[0] = inner_prod(rKin.A1, rKin.dud1) + 0.5 * std::sqrt(norm_2(rKin.dud1));
+        rKin.metricChange[1] = inner_prod(rKin.A2, rKin.dud2) + 0.5 * std::sqrt(norm_2(rKin.dud2));
```

Comment:
Hey, out of curiosity, why did you change this? is this failing?

Reply:
It was failing for me in win, but seems that is not giving problems anywhere else, so probably it was a problem in my side